test/unit/definitions/divorce/CaseTypeTab.test.js

```diff
@@ -1,11 +1,26 @@
 const expect = require('chai').expect;
 const assert = require('chai').assert;
 const { uniq, uniqWith, map, filter } = require('lodash');
+const { loadAllFiles, sortCaseTypeTabs } = require('../../utils/utils');
 
-const caseTypeTab = Object.assign(require('definitions/divorce/json/CaseTypeTab/CaseTypeTab'), {});
-const caseField = Object.assign(require('definitions/divorce/json/CaseField/CaseField'), {});
+const getAllCaseTypeTabDefinitions = loadAllFiles('CaseTypeTab');
+const caseTypeTab = getAllCaseTypeTabDefinitions(
+  [
+    'CaseTypeTab',
+    'CaseTypeTab-prod',
+    'CaseTypeTab-deemed-and-dispensed-nonprod'
+  ]);
 
-const tabIds = uniq(map(caseTypeTab, 'TabID'));
+const getAllCaseFieldDefinitions = loadAllFiles('CaseField');
+const caseField = getAllCaseFieldDefinitions(
+  [
+    'CaseField',
+    'CaseField-prod',
+    'CaseField-deemed-and-dispensed-nonprod'
+  ]);
+
+const sortedCaseTabs = sortCaseTypeTabs(caseTypeTab);
+const tabIds = uniq(map(sortedCaseTabs, 'TabID'));
 
```

Comment:
case tab ids needs to be in ascending order for an exiting for a test case to pass